docs/extending/custom_bulk_actions.md

```diff
@@ -30,7 +30,7 @@ The attributes are as follows:
 
 -   `display_name` - The label that will be displayed on the button in the user interface
 -   `aria_label` - The `aria-label` attribute that will be applied to the button in the user interface
--   `action_type` - A unique identifier for the action. Will be required in the url for bulk actions
+-   `action_type` - A unique identifier for the action will be required in the url for bulk actions
```

Comment:
Maybe we should be a bit clearer here, the first part is the description and the second part is a note about it being required in some cases.

Also, none of these lines are written as sentences so changing this one to a sentence feels a bit off.

Here is a suggestion

```suggestion
-   `action_type` - A unique identifier for the action (required in the URL for bulk actions)
```